WORKSPACE

```diff
@@ -2,7 +2,9 @@
 # `@cockroach//...`.
 workspace(
     name = "cockroach",
-    managed_directories = {"@npm": ["node_modules"]},
+    managed_directories = {
+       "@npm": ["pkg/ui/node_modules"]
```

Comment:
Do the other `node_modules` directories (`pkg/ui/workspaces/cluster-ui/node_modules`, `pkg/ui/workspaces/db-console/node_modules`) need to be in this list as well?

Reply:
Done, thanks!